front/cypress/e2e/email_consent.cy.ts

```diff
@@ -46,7 +46,6 @@ describe('email consent', () => {
 
     cy.get('#e2e-custom-email-container iframe').then(($iframe) => {
       const $body = $iframe.contents().find('body');
-      cy.wrap($body).find('#e2e-email-settings-page');
       cy.wrap($body).find('#e2e-consent-form');
```

Comment:
Since checking #e2e-consent-form implicitly checks #e2e-email-settings-page, we can just remove the first check. For some reason the first check is super flaky on CI (maybe because it's the outermost div in the iframe or something - I'm not too sure).